docs/home/howto/LOGGING.md

```diff
@@ -56,7 +56,7 @@ specification:
     logs: /var/log/elasticsearch
 ```
 
-## How to manage Opendistro for Elasticsearch data
+## Opendistro for Elasticsearch data management
```

Comment:
If you want to change this header you have change reference here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md